MoltenVK/MoltenVK/Commands/MVKCmdDraw.mm

```diff
@@ -723,12 +723,17 @@
                         [cmdEncoder->_mtlRenderEncoder setTessellationFactorBuffer: tcLevelBuff->_mtlBuffer
                                                                             offset: tcLevelBuff->_offset
                                                                     instanceStride: 0];
+#if MVK_MACOS_OR_IOS
                         [cmdEncoder->_mtlRenderEncoder drawPatches: outControlPointCount
                                                   patchIndexBuffer: nil
                                             patchIndexBufferOffset: 0
                                                     indirectBuffer: tcIndirectBuff->_mtlBuffer
                                               indirectBufferOffset: mtlTCIndBuffOfst];
-                        mtlTCIndBuffOfst += sizeof(MTLDrawPatchIndirectArguments);
+#endif
```

Comment:
A side-effect of which is that tvOS doesn't support it at all and this code needs to be conditionally compiled to avoid build warnings. No doubt we'll be able to remove this in a future tvOS release.

WRT the feature bit itself, how are you currently handling the GPU fam 5 restriction?

Reply:
Uhh... we're not? We don't have any products on iOS right now.

Reply:
> Uhh... we're not? We don't have any products on iOS right now.

Sorry...I was asking if MoltenVK guards against this currently. Or is that the feature bit you're indicating we need to add?

Reply:
Yes, that's the feature bit we need to add. This was an oversight on my part when I added tessellation support.

Reply:
Added issue #937.